Humbertino_Avilez/Day4/Gradle_Hackerrank/NewGradle/lib/src/main/java/hackerrank/JavaInToString.java

```diff
@@ -0,0 +1,25 @@
+package hackerrank;
+
+import java.util.Scanner;
+
+import org.apache.log4j.Logger;
+
+import com.sun.tools.javac.Main;
+
+public class JavaInToString {
+	private static Logger javaLog = Logger.getLogger(Main.class);
```

Comment:
I think logger var should be final
https://rules.sonarsource.com/java/tag/convention/RSPEC-1312